src/Router.sol

```diff
@@ -444,26 +467,19 @@ contract Router is IRouter, Ownable, EIP712 {
         emit DelegationNonceInvalidation(msg.sender, delegatee, newNonce, oldNonce);
     }
 
-    /// @notice Set the fee collector address that collects fees from each user's agent by owner
-    /// @param feeCollector_ The fee collector address
-    function setFeeCollector(address feeCollector_) public onlyOwner {
-        if (feeCollector_ == address(0)) revert InvalidFeeCollector();
-        feeCollector = feeCollector_;
-        emit FeeCollectorSet(feeCollector_);
-    }
-
-    /// @notice Set the pauser address that can pause `execute` and `executeWithSignerFee` by owner
-    /// @param pauser_ The pauser address
-    function setPauser(address pauser_) public onlyOwner {
-        if (pauser_ == address(0)) revert InvalidNewPauser();
-        pauser = pauser_;
-        emit PauserSet(pauser_);
-    }
+    /// @notice Invalidate nonces for an execution
+    /// @param newNonce The new nonce to set. Invalidates all nonces less than it
+    /// @dev Can't invalidate more than 2**16 nonces per transaction
+    function invalidateExecutionNonces(uint256 newNonce) external {
+        uint256 oldNonce = executionNonces[msg.sender];
+        if (newNonce <= oldNonce) revert InvalidNonce();
+        // Limit the amount of nonces that can be invalidated in one transaction.
+        unchecked {
+            uint256 delta = newNonce - oldNonce;
+            if (delta > type(uint16).max) revert ExcessiveInvalidation();
```

Comment:
just found we may be able to skip `uint256 delta = newNonce - oldNonce` and use `if (newNonce - oldNonce > type(uint16).max)` instead?

Reply:
In fact it's from [permit2](https://github.com/Uniswap/permit2/blob/main/src/AllowanceTransfer.sol#L120C20-L120C25). But yes, we can do that.